src/stubs/git-pre-commit

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
 
-git diff --diff-filter=d --name-only HEAD | grep ".php$" | xargs ./vendor/bin/phpcs --standard=phpcs.xml
+CHANGED_PHP=$(git diff --diff-filter=d --name-only HEAD | grep ".php$")
+
+if [ -n "$CHANGED_PHP" ] ; then
+	./vendor/bin/phpcs --standard=phpcs.xml $CHANGED_PHP
+fi
```

Comment:
Great! It's a bug.

I have two suggestions:

1. shell style: https://github.com/icy/bash-coding-style#variable-names

> If a variable can be changed from its parent environment, it should be in uppercase; e.g, THIS_IS_A_USER_VARIABLE.
> Other variables are in lowercase

2. expansibility for phpmd

```suggestion
files=$(git diff --diff-filter=d --name-only HEAD | grep '.php$')
for file in $files; do
  ./vendor/bin/phpcs --standard=phpcs.xml "$file"
  # ./vendor/bin/phpmd "$file" text phpmd.xml --exclude vendor
done
# ./vendor/bin/phpunit tests/
```

Reply:
Thanks for suggestions! My brain had something wrong on Friday so var had named uppercase 😅
Improved at 2b3e2dd

I'm not familiar with phpmd but i think the loop mainly for phpmd? If we use phpcs in loop, it will stop while finding error. This cause user commit continuously unless all changed PHP files pass phpcs, maybe. Any idea?

Reply:
phpmd can scan something which phpcs can not find. such as:

> The class Foo has 11 public methods. Consider refactoring Foo to keep number of public methods under 10.
> The method foo() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.